protocol/x/perpetuals/keeper/perpetual.go

```diff
@@ -293,7 +293,6 @@ func (k Keeper) processPremiumVotesIntoSamples(
 					metrics.PerpetualId,
 					int(perp.GetId()),
 				),
-				// TODO(DEC-1071): Add epoch number as label.
```

Comment:
We decided not to add epoch number or anything that has unbonded cardinality as labels.